src/app/share/views/SharedLinksView/SharedView.tsx

```diff
@@ -89,17 +93,20 @@ function SharedView(props: SharedViewProps): JSX.Element {
   const folderUUID = urlParams.get('folderuuid');
 
   const itemToRename = useAppSelector((state: RootState) => state.storage.itemToRename);
-  const isFileViewerOpen = useAppSelector((state: RootState) => state.ui.isFileViewerOpen);
 
   const [hasMoreItems, setHasMoreItems] = useState<boolean>(true);
   const [hasMoreFolders, setHasMoreFolders] = useState<boolean>(true);
   const [hasMoreRootFolders, setHasMoreRootFolders] = useState<boolean>(true);
   const [page, setPage] = useState<number>(0);
   const [orderBy, setOrderBy] = useState<OrderBy>();
   const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [isStopSharingDialogLoading, setIsStopSharingDialogLoading] = useState<boolean>(false);
   const [selectedItems, setSelectedItems] = useState<any[]>([]);
   const [shareItems, setShareItems] = useState<AdvancedSharedItem[]>([]);
+  const [showStopSharingConfirmation, setShowStopSharingConfirmation] = useState(false);
   const [editNameItem, setEditNameItem] = useState<DriveItemData>();
+  const [itemToView, setItemToView] = useState<any>();
```

Comment:
Avoid the use of `any` type as much as possible, add the correct type of the element